### DISCUSSION TITLE
Google CloudIOT

### DISCUSSION THREAD

**Discussion by MikeAtETI** [dotnet/MQTTnet#1249] [Q&A]
Does anyone have a good example of running this against Google CloudIOT's MQTT gateway?

**Comment by sheindel** [comment]
Hi Mike. Google does provide some [IoT core dotnet samples](https://github.com/GoogleCloudPlatform/dotnet-docs-samples/tree/master/iot/api), but if you still would like an example with this library specifically, I recently implemented the connection.
